### ISSUE TITLE
Not sure how to "ensure that libreadstat.dylib is in Julia's load path."

### ISSUE THREAD

**Issue by pnvolkmar** [WizardMac/DataRead.jl#3]
I've downloaded the Readstat-master directory, but don't know how to get from there to getting the libreadstat.dylib file (or, for that matter, getting it into Julia's load path).

**Comment by diogocp** [comment]
You have to compile [WizardMac/ReadStat](https://github.com/WizardMac/ReadStat).

I'm still stuck trying to get Julia to load it.

**Comment by pnvolkmar** [comment]
I installed Ragel and it worked.

**Comment by JessicaYK** [comment]
Hi all,

This is a basic question, but how can I compile WizardMac/ReadStat? Is it possble to compile it on Windows? How does Ragel work? Any help will be appreciated!

**Comment by diogocp** [comment]
@JessicaYK It is possible to compile ReadStat on Windows with Cygwin or MSYS2, but it is not straightforward and in the end I could not get it to work with Julia. Ragel is a compile-time dependency of ReadStat.

If you decide to try it, you will need to make some changes to the ReadStat makefile. Something like this: https://github.com/diogocp/ReadStat/commit/17f8276bacbd91e48018284fbbedfae472fd7eb4

Basically the steps should be: install MSYS2, install a toolchain, compile and install Ragel, patch the ReadStat makefile and compile it.

**Comment by JessicaYK** [comment]
@diogocp Thank  you so much for your quick and detailed comment. I will follow your instructions and let you know if it works!
